src/mac.js

```diff
@@ -288,7 +288,7 @@ class MacApp extends App {
   }
 
   async signAppIfSpecified () {
-    const osxSignOpt = this.opts.osxSign
+    const osxSignOpt = this.opts['osx-sign'] || this.opts.osxSign
```

Comment:
This is in the wrong place, it should probably be in `src/cli.js`.

Reply:
@malept OK, I made the change and added the test.